docs/process-services-cloud/directives/unclaim-task-cloud.directive.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: Unclaim Task Cloud Directive
+Added: v3.1.0
+Status: Experimental
+Last reviewed: 2019-03-25
+---
+
+# [Unclaim Task Cloud directive](../../../lib/process-services-cloud/src/lib/task/directives/unclaim-task-cloud.directive.ts "Defined in unclaim-task-cloud.directive.ts")
+
+Unclaims a task
+
+## Basic Usage
+
+```html
+<button adf-cloud-unclaim-task [appName]="appName" [taskId]="taskId" (success)="onTaskUnclaimed()">Unclaim</button>
+```
+
+## Class members
+
+### Properties
+
+| Name | Type | Default value | Description |
+| ---- | ---- | ------------- | ----------- |
+| appName | `string` | "" | (Required) The name of the application. |
+| taskId | `string` |  | (Required) The id of the task. |
```

Comment:
Can you add a default value for the taskId?

Reply:
Do we really need to add default here?